resources/providers/PingIdentity/Provider.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace SocialiteProviders\PingIdentity;
+
+use GuzzleHttp\RequestOptions;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
+{
+    public const IDENTIFIER = 'PINGIDENTITY';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopes = ['email', 'family_name', 'given_name'];
```

Comment:
It would be better to put native PHP annotations (and PHPDoc comments, if necessary) so PHPStan can pick up on the type.  As far as I am aware, `@inheritdoc`, is not interpreted by PHPStan.